docs/quick_tutorial/debugtoolbar.rst

```diff
@@ -65,7 +65,11 @@ Steps
         $VENV/bin/pserve development.ini --reload
 
 #.  Open http://localhost:6543/ in your browser.
-    See the handy toolbar on the right. ``note: the Response object should contain <body> html or it won't be shown unless an error is introduced.``
+    See the handy toolbar on the right. 
+    
+      
+      .. note:: 
+        the Response object should contain ``<body></body>`` tag , so if you have changed this in previous steps of the tutorial make sure that you include the it         in order for this to work.
```

Comment:
This is a duplicate of debugtoolbar documentation, and we already link to it in the Quick Tutorial under [Usage](https://docs.pylonsproject.org/projects/pyramid_debugtoolbar/en/latest/#usage). I'm sorry, but I cannot approve this change.

Reply:
this is a **quick tutorial** not a documentation , the documentation is comprehensive but who is doing a **quick tutorial**  then goes to a specific documentation about debugtoolbar when you have nothing to debug and hasn't even started using the framework ? you know where I am going with this ?

Reply:
this specific change doesn't have to be merged , the point is to make the documentation more easy to follow and account for such a change. 

that is the point of opening this PR.